alerts/homematicip_cloud.markdown

```diff
@@ -0,0 +1,31 @@
+---
+title: HomematicIP (EQ-3) blocks public IP addresses, if access to the Cloud is too frequent.
+created: 2020-12-20 12:00:00
+integrations:
+  - homematicip_cloud
+packages:
+  - homematicip
+homeassistant: ">0.7"
+---
+
+Since Dec 1, 2020 HomematicIP (EQ-3) has started to **block public IP addresses** of Home Assistant installations if they access the Homematic IP Cloud **too often**. In this case, the Homematic IP Cloud can only be reached from a cellular network. 
+Further plans to completely block the non-partner apps from the Homematic IP Cloud cannot be ruled out.
```

Comment:
```suggestion
Since Dec 1, 2020, HomematicIP (EQ-3) has started to **block public IP addresses** of Home Assistant installations, if they access the Homematic IP Cloud **too often**. In this case, the Homematic IP Cloud can only be reached from a cellular network.
Further plans to completely block the non-partner apps from the Homematic IP Cloud cannot be ruled out.
```